python/ray/experimental/state/api.py

```diff
@@ -479,17 +479,30 @@ def summary(
         return summary_api_response["result"]["node_id_to_summary"]
 
 
-"""
-Convenient Methods for get_<RESOURCE> by id
-"""
-
-
 def get_actor(
     id: str,
     address: Optional[str] = None,
     timeout: int = DEFAULT_RPC_TIMEOUT,
     _explain: bool = False,
 ) -> Optional[ActorState]:
+    """Get an actor by id.
+
+    Args:
+        id: Id of the actor
+        address: The IP address and port of the head node. Defaults to
+            `http://localhost:8265` if not set.
```

Comment:
Is this true? I thought it is 6379

Reply:
I guess I should call it "API server url" to be consistent with others.